### PR TITLE
docs(zh): add translation for `@pinia/nuxt.ModuleOptions` to avoid `404`

### PR DESCRIPTION
**Pull request by utakotoba**
- add translation for `@pinia/nuxt.ModuleOptions` to avoid `404`
- with some other fix translation in `Module: pinia`

### CODE REVIEW

File: packages/docs/zh/api/interfaces/pinia_nuxt.ModuleOptions.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+]
+```
+
+---
+
+### disableVuex %{#Properties-disableVuex}%
+
+• `Optional` **disableVuex**: `boolean`
+
+默认情况下，Pinia 会禁用 Vuex，将此选项设置为 `false` 以避免禁用 Vuex，并在 Nuxt 2 中同时使用 Pinia 和 Vuex。
```
Comment (KimYangOfCat) [inline packages/docs/zh/api/interfaces/pinia_nuxt.ModuleOptions.md]:
```suggestion
默认情况下，Pinia 会禁用 Vuex，将此选项设置为 `false` 可启用 Vuex，然后便可同时使用 Pinia 和 Vuex（仅在 Nuxt 2 中支持）。
```

File: packages/docs/zh/api/modules/pinia.md [hunk 2]
```diff
@@ -458,7 +458,7 @@ ___
 
 ▸ (...`args`): `any`
 
-Generic type for a function that can infer arguments and return type
+可以推断参数和返回类型的函数的通用类型。
```
Comment (KimYangOfCat) [inline packages/docs/zh/api/modules/pinia.md]:
```suggestion
可以推断参数和返回值类型的函数通用类型。
```

**Comment by utakotoba** [comment]
By the way, i think the repo should have the formatter config for the Markdown file while many IDE will reformat them lead to  make additional diff.

**Comment by KimYangOfCat** [comment]
@ifshizuku  The formatting of the contents of markdown files is not relevant to this PR, so if you want to add relevant configurations, please create a new issue to discuss  or a new PR to provide a viable solution. So can you remove the changes  about formatting associated with the last commit?

**Comment by utakotoba** [comment]
> @ifshizuku  The formatting of the contents of markdown files is not relevant to this PR, so if you want to add relevant configurations, please create a new issue to discuss  or a new PR to provide a viable solution. So can you remove the changes  about formatting associated with the last commit?

@KimYangOfCat I'll do it later

**Comment by utakotoba** [comment]
@KimYangOfCat The L1202 of the module pinia seem to be the same but it also show in diff, please check it again, thanks. or should i rebase the whole branch to remove the format issued commit from history... maybe reopen a PR is better, sorry for that.

**Comment by KimYangOfCat** [comment]
it‘s OK，I checked the Files Changed of this PR，I think It’s OK，the rebase shouldn't be a problem if this PR will be Squash Merged. 
Thank you for your contribution. @ifshizuku 
so LGTM @posva

**Pull request merged**